SofaKernel/modules/SofaBaseTopology/EdgeSetTopologyContainer.cpp

```diff
@@ -409,10 +395,8 @@ const EdgeSetTopologyContainer::VecEdgeID EdgeSetTopologyContainer::getElementAr
 
     if (!hasEdgesAroundVertex())
     {
-		if(CHECK_TOPOLOGY)
-			msg_warning() << "Edge vertex shell array is empty.";
-
-        createEdgesAroundVertexArray();
+    	msg_warning() << "getElementAroundElement: Edge vertex shell array is empty. Be sure to call createEdgesAroundVertexArray first.";
```

Comment:
This cannot be a msg_warning as the message is for developper only because the provided solution imply checking source code.